### PR TITLE
change Statsv2 endpoint

### PR DESCRIPTION
**Pull request by elreco**
#12 fix statsv2 endpoint

### CODE REVIEW

File: resources/Endpoints/index.js [hunk 1]
```diff
@@ -1,40 +1,40 @@
...
-  ORDER_PURCHASE: 'https://orderprocessor-public-service-ecomprod01.ol.epicgames.com/orderprocessor/api/shared/accounts',
-  CAPTCHA_PURCHASE: 'https://launcher-website-prod07.ol.epicgames.com/purchase',
-
-  // ACCOUNT
-  ACCOUNT: 'https://account-public-service-prod03.ol.epicgames.com/account/api/public/account',
-  ACCOUNT_BY_NAME: 'https://account-public-service-prod03.ol.epicgames.com/account/api/public/account/displayName',
-
-  // FN ENDPOINTS
-  STATS_BR_V1: 'https://fortnite-public-service-prod11.ol.epicgames.com/fortnite/api/stats/accountId',
-  STATS_BR_V2: 'https://fortnite-public-service-prod11.ol.epicgames.com/fortnite/api/statsv2/account',
```
Comment (elreco) [inline resources/Endpoints/index.js:23]: here

**Comment by iXyles** [comment]
Can you please change back to LF line breaking?

**Comment by elreco** [comment]
Is that okay?

**Review by iXyles — APPROVED** [review]
Thanks.

**Comment by iXyles** [comment]
Pushing to npm later today.

**Comment by elreco** [comment]
Ok thanks !

**Pull request merged**